test/subscription.go

```diff
@@ -96,3 +97,24 @@ func CreateOperatorGroup(ctx *Context, name, namespace string) (*operatorsv1.Ope
 			},
 		}, metav1.CreateOptions{})
 }
+
+func GetCatalogSourceWithCSVInChannel(ctx *Context, csv, channel, namespace string) (string, error) {
```

Comment:
it might still be useful to prefer the defined CatalogSource, in case there are multiple of them containing the CSV (e.g. in case we still build the CI bundle with previous versions as well, we may want to prefer the `serverless-operator` if that one has the CSV, and use any other if the preferred one does not contain it?

Reply:
OK